index.html

```diff
@@ -74,5 +76,179 @@ <h3>   I am a Software Developer </h3>
 
       </div>
     </div>
+<section class="work-section">
+  <header class="work-section-heading">
+    <h2>My Recent Works</h2>
+    <div class="line"></div>
+    <!-- <hr class="hr"> -->
+  </header>
+  <div class="work-section-image">
+    <img class="img" src="./Images/image1.jpg" alt="">
+  </div>
+  <section class="work-section-texts">
+      <h3 class="work-section-text1">Multi-Post Stories</h3>
+      <p class="paragraph">
+        A daily selection of privately personalized reads;
+         no accounts or sign-ups required. has been the industry's
+          standard dummy text ever since the 1500s,
+         when an unknown printer took a standard dummy text.
+      </p>
+      <div class="work-section-button">
+        <button type="button" class="btn1"><span>Css</span></button>
```

Comment:
- [ ] For the technologies, is not required to have a `:hover` state, please get rid of this style. According to your figma template, all technologies must look like the following pictures, depending on the case:
![image](https://user-images.githubusercontent.com/52635378/164579966-b820bd6a-2486-445a-9eff-0bb81d439411.png)
![image](https://user-images.githubusercontent.com/52635378/164579987-32dba9f7-5908-4213-b895-0849971c52c5.png)